api/src/repositories/submission-repository.ts

```diff
@@ -46,6 +46,10 @@ export interface ISubmissionModel {
   revision_count: number;
 }
 
+export interface ISubmissionModelWithStatus extends ISubmissionModel {
```

Comment:
Is the reason for extending ISubmission model that the status comes from a different table?

Reply:
Yeah, I think that function is returning all of the submission rows + the 1 status row from the other table

Reply:
That is correct